### PR TITLE
GROWTH-7169 disallow the /similar/ all for locales

### PR DESCRIPTION
**Pull request by vtnk**
### Description
Google doesn't seem to be excluding anything from the en-gb robots.txt, and there are a lot of /similar/ pages that are not being crawled / indexed.

We want to only keep the robots.txt file on the root domain (core site), and remove https://www.1stdibs.com/en-gb/robots.txt. Anything that we want to be disallowed from all directories needs to work at the root robots.txt

Implementation:

We will want to explicitly disallow the /similar/ segment for each locale, which would make the rule : 

Disallow: /fr/similar/
Disallow: /en-gb/similar/
Disallow: /de/similar/
Disallow: /similar/

### Ticket
https://1stdibs.atlassian.net/browse/GROWTH-7169

### CODE REVIEW

File: www.1stdibs.com/robots.txt [hunk 1]
```diff
@@ -29,6 +29,9 @@ Disallow: /recognized-dealers/fashion/
 Disallow: /more-from-seller/
 Disallow: /mobile/
 Disallow: /resp/
```
Comment (gedas-monginas) [inline www.1stdibs.com/robots.txt:31]: base on:
> This should be done for every Disallow rule that is for a segment, query param rules such as Disallow: /*q=* do not need to be modified. 

I think same 3 additional local records should be added for all `Disallow` rules above ☝️  and below 👇 
just we don't need to change `/*q=*` and probably `.php` files 🤔

Comment (vtnk) [reply www.1stdibs.com/robots.txt:31]: I cleared this question with David Hu and he said to modify only /similar/ urls. @nameofname can you confirm this?

Comment (gedas-monginas) [reply www.1stdibs.com/robots.txt:31]: that doesn't make much sense to me 🤷

Comment (nameofname) [reply www.1stdibs.com/robots.txt:31]: That's right, it doesn't make sense. Pinging David to see if he'll respond via slack

**Comment by vtnk** [comment]
This is quite repetitive and large change, however I didn't find other syntax for robots.txt except like `/*/similar/` which would be too broad.

File: www.1stdibs.com/robots.txt [hunk 2]
```diff
@@ -3,32 +3,113 @@ Allow: /soa/graphql/$
...
+Disallow: /en-gb/notfound.html
+Disallow: /de/notfound.html
 Disallow: /notfound.html
+Disallow: /fr/privacypolicy.php
+Disallow: /en-gb/privacypolicy.php
+Disallow: /de/privacypolicy.php
 Disallow: /privacypolicy.php
+Disallow: /fr/useragreement.php
+Disallow: /en-gb/useragreement.php
+Disallow: /de/useragreement.php
```
Comment (gedas-monginas) [inline www.1stdibs.com/robots.txt:60]: @nameofname should we care about .php files?
as there should be no links to `/de/useragreement.php` for example, I wanna believe we don't really need to add these rules 🤔

Comment (nameofname) [reply www.1stdibs.com/robots.txt:60]: @gedas-monginas I agree, I don't see any reason to add it. Let's ask SEO to inspect the changes before we deploy.

Comment (gedas-monginas) [reply www.1stdibs.com/robots.txt:60]: I doubt they gonna care about tidiness of this file 😅

Comment (vtnk) [reply www.1stdibs.com/robots.txt]: Removed changes for `.php` lines. Should we assume SEO is happy about changes and we can merge this PR? 😃

**Review by gedas-monginas — APPROVED** [review]

**Pull request merged**